src/app/(proper_react)/redesign/(authenticated)/user/welcome/Onboarding.test.tsx

```diff
@@ -112,9 +112,7 @@ it("form input elements have invalid state if left empty on step 2", async () =>
   expect(firstNameInput.getAttribute("aria-invalid")).toBe(null);
   expect(lastNameInput.getAttribute("aria-invalid")).toBe(null);
   expect(locationInput.getAttribute("aria-invalid")).toBe(null);
-  if (dateInput) {
-    expect(dateInput.getAttribute("aria-invalid")).toBe(null);
-  }
+  expect(dateInput?.getAttribute("aria-invalid")).toBe(null);
```

Comment:
Conditional `expect`s wouldn't have failed the the `dateInput` inadvertently didn't exist.